phpBB/viewtopic.php

```diff
@@ -263,6 +263,20 @@
 
 $forum_id = (int) $topic_data['forum_id'];
 
+/**
+ * Modify the forum ID to handle the correct display of viewtopic if needed
+ *
+ * @event core.viewtopic_modify_forum_id
+ * @var string	forum_id		forum ID
+ * @var array	topic_data		array of topic's data
+ * @since 3.2.4
```

Comment:
We will not add any more events to the 3.2.4 release. Please change this to `3.2.5-RC1`, thanks.

Reply:
Will do